### PR TITLE
[MIG] stock_request_mrp: Migration to 15.0

### PR DESCRIPTION
**Pull request by augusto-weiss**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
LGTM

**Comment by rousseldenis** [comment]
@augusto-weiss A better PR tittle would help reviewers. Thanks

Like `[MIG] stock_request_mrp: Migration to 15.0`

**Comment by rousseldenis** [comment]
@augusto-weiss Could you check tests?

File: stock_request_mrp/tests/test_stock_request_mrp.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            }
+        )
+
+    def _update_qty_in_location(self, location, product, quantity):
+        self.env["stock.quant"]._update_available_quantity(product, location, quantity)
+
+    def _create_user(self, name, group_ids, company_ids):
+        return (
+            self.env["res.users"]
+            .with_context({"no_reset_password": True})
```
Comment (nicomacr) [inline stock_request_mrp/tests/test_stock_request_mrp.py]:
```suggestion
            .with_context(no_reset_password=True)
```

File: stock_request_mrp/tests/test_stock_request_mrp.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        for raw_mat in raw_materials:
+            self.env["mrp.bom.line"].create(
+                {"bom_id": bom.id, "product_id": raw_mat.id, "product_qty": 1}
+            )
+
+        return bom
+
+    def _produce(self, mo, qty=0.0):
+        wiz = Form(
+            self.produce_wiz.with_context({"active_id": mo.id, "active_ids": [mo.id]})
```
Comment (nicomacr) [inline stock_request_mrp/tests/test_stock_request_mrp.py]:
```suggestion
            self.produce_wiz.with_context(active_id=mo.id, active_ids=[mo.id])
```

**Review by nicomacr — CHANGES_REQUESTED** [review]
@JoelZilli check these points below please

**Comment by nicomacr** [comment]
@rousseldenis could you check now this PR?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**